frontend/lib/util/get_application_url.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:ehrenamtskarte/build_config/build_config.dart';
+import 'package:ehrenamtskarte/configuration/definitions.dart';
+import 'package:ehrenamtskarte/constants/project_ids.dart';
+import 'package:ehrenamtskarte/identification/util/card_info_utils.dart';
+import 'package:ehrenamtskarte/proto/card.pb.dart';
+
+String getApplicationUrl(ApplicationUrl applicationUrl, bool isStagingEnabled) {
```

Comment:
:wrench: Why not just pass the build context and handle everything in these two methods? Would make clearer code  and separate responsibilities better imo.
```suggestion
String getApplicationUrl(BuildContext context) {
  final isStagingEnabled = Provider.of<SettingsModel>(context, listen: false).enableStaging;
  final applicationUrl = buildConfig.applicationUrl;
```

Reply:
because it makes testing way more difficult to mock the whole  build context.

Reply:
added this in last commit